### PR TITLE
Add Loki documentation

### PR DESCRIPTION
**Pull request by DebakelOrakel**
## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `how-to`, `tutorial`, `reference`, `explanation`, `cicd`, `dependency`
      as they show up in the changelog
- [ ] Link this PR to related issues if applicable.

### CODE REVIEW

**Review by schemen — APPROVED** [review]

File: docs/modules/ROOT/pages/tutorials/logging/getting-started.adoc [hunk 1]
```diff
@@ -0,0 +1,99 @@
+= Getting started with logging: Lokistack
+
+This tutorial explains how to view the logs of an application on Openshift.
```
Comment (tobru) [inline docs/modules/ROOT/pages/tutorials/logging/getting-started.adoc:3]:
```suggestion
This tutorial explains how to view the logs of an application on the OpenShift Web Console with the LokiStack.
```

File: docs/modules/ROOT/pages/tutorials/logging/getting-started.adoc [hunk 2]
```diff
@@ -0,0 +1,99 @@
+= Getting started with logging: Lokistack
```
Comment (tobru) [inline docs/modules/ROOT/pages/tutorials/logging/getting-started.adoc:1]:
```suggestion
= Getting started with logging
```

**Review by tobru — COMMENTED** [review]
Do you want to add some deprecation notes to the Kibana part? Or already completely remove it?

**Comment by tobru** [comment]
> Yes, I'd go for completly removing it.

Then I suggest: Remove the Kibana page, add an [URL redirect](https://docs.antora.org/antora/latest/page/page-aliases/) from Kibana to LokiStack and mention the deprection on the LokiStack docs page with an appropriate admonition.

**Review by tobru — APPROVED** [review]

**Pull request merged**